unit-11/javascrip/main.js

```diff
@@ -0,0 +1,103 @@
+// Lesson 1
+function getArray (array, n) {
+  const newArray = []
+  for (let i = 0; i < n; i++) {
+    newArray.push(array)
+  }
+  return newArray
+}
+// lesson 2
+function getArrayReverse (...array) {
+  const arrayReverse = []
+  for (let i = 0; i < array.length; i++) {
+    arrayReverse.unshift(array[i])
+  }
+  return arrayReverse
+}
+// lesson 3
+const array = [0, 1, false, 2, undefined, '', 3, null]
+
+const deleteValueFalse = array.filter( function(array) {
+  return array
+})
+
+// lesson 4
+const data = [['a', 1], ['b', 2], ['c', 33]]
+function getObjectFromArray (array) {
+  const objectFromArray = {}
+  for (let  i = 0;  i < array.length;  i++) {
+    let value = `${array[i].slice(0, 2)}`
+    let standardValue = value.replace(',', ': ')
+    let key = standardValue.slice(0, 1)
+    let valueObj = standardValue.slice(3)
+    objectFromArray[key] = Number(valueObj)
+  }
+  return objectFromArray
+}
+// lesson 5
+function getArraysAscending (...array) {
```

Comment:
```suggestion
function getArraysAscending (array) {
```

Viết còn ko chịu chạy thử xem đúng hay ko
Hay lại đi copy chỗ nào